### PR TITLE
Add test files

### PR DESCRIPTION
**Pull request by academey**

### CODE REVIEW

File: app/__mocks__/currentUser.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "affiliation": "eeee",
+  "articleCount": 0,
+  "commentCount": 0,
+  "email": "honeyjasdds@postech.ac.kr",
+  "emailVerified": false,
+  "id": 39,
+  "isLoggedIn": true,
+  "major": null,
+  "name": "박현준",
```
Comment (TylorShin) [inline app/__mocks__/currentUser.json]: English will be better.

File: app/components/auth/signIn/__tests__/actions_spec.tsx [hunk 2]
```diff
@@ -68,4 +70,153 @@ describe("sign in actions", () => {
...
+      const actions = store.getActions();
+      expect(actions[2]).toEqual(push("/"));
+    });
+
+    it("should return push action to home page if it doesn't exist", async () => {
+      await store.dispatch(Actions.getAuthorizeCode(mockCode, mockVendor, null));
+      const actions = store.getActions();
+      const mockRecordifiedUser = recordify(initialMember);
+
+      expect(JSON.stringify(actions[3])).toEqual(
```
Comment (TylorShin) [inline app/components/auth/signIn/__tests__/actions_spec.tsx]: This is ugly too.

File: app/model/__tests__/comment_spec.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        expect(recordifyComment().toString()).toContain("Record");
+      });
+
+      it("should return same value with initial state", () => {
+        expect(recordifyComment().toJS()).toEqual(initialComment);
+      });
+    });
+
+    describe("when there are params", () => {
+      const mockId = 233232;
```
Comment (TylorShin) [inline app/model/__tests__/comment_spec.tsx]: You can make an object.

File: app/model/__tests__/paper_spec.tsx [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    it("should return initial state when there is no param", () => {
+      expect(recordifyPaper().toJS()).toEqual(initialPaper);
+    });
+
+    it("should return recordified state when there is no param", () => {
+      expect(recordifyPaper().toString()).toContain("Record");
+    });
+
+    describe("when there are params", () => {
+      let mockUserObject: IPaper;
```
Comment (TylorShin) [inline app/model/__tests__/paper_spec.tsx]: You can make an object here too.

File: app/components/articleSearch/__tests__/records_spec.tsx [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+        expect(state.toString()).toContain("Record");
+      });
+
+      it("should return initial state", () => {
+        expect(state).toEqual(ARTICLE_SEARCH_INITIAL_STATE);
+      });
+    });
+
+    describe("when there is normal js params", () => {
+      const mockIsLoading = false;
```
[the remark this answers is not in the document]
Comment (academey) [reply app/components/articleSearch/__tests__/records_spec.tsx]: I don't understand why we should do like that.

File: app/components/auth/__tests__/actions_spec.tsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  let store: any;
+
+  beforeEach(() => {
+    store = generateMockStore({});
+    store.clearActions();
+  });
+
+  describe("signOut action", () => {
+    it("should return AUTH_SUCCEEDED_TO_SIGN_OUT action", async () => {
+      window.confirm = jest.fn(() => true);
```
[the remark this answers is not in the document]
Comment (academey) [reply app/components/auth/__tests__/actions_spec.tsx]: Ok I will replace that

File: app/components/articleSearch/__tests__/records_spec.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (academey) [reply app/components/articleSearch/__tests__/records_spec.tsx]: I think we have to initialize before each test

**Pull request merged**